source/sigv4.c

```diff
@@ -1356,15 +1377,15 @@ static SigV4Status_t generateCredentialScope( const SigV4Parameters_t * pSigV4Pa
         assert( ( value != NULL ) && ( index < valLen ) );
 
         /* Only trim spaces. */
-        if( isspace( value[ index ] ) )
+        if( isspace( value[ index ] ) != 0U )
```

Comment:
This will have its own implementation in a separate PR

Reply:
True, I just wanted wanted one of the warnings to go away. It'll be overwritten later anyway